livekit-rtc/livekit/rtc/participant.py

```diff
@@ -107,6 +107,11 @@ def kind(self) -> proto_participant.ParticipantKind.ValueType:
         """Participant's kind (e.g., regular participant, ingress, egress, sip, agent)."""
         return self._info.kind
 
+    @property
+    def disconnect_reason(self) -> proto_participant.DisconnectReason.ValueType:
```

Comment:
Maybe we should change this return `proto_participant.DisconnectReason.ValueType | None`